src/components/Input/DropDown/DropDownGroup.styles.js

```diff
@@ -131,7 +131,9 @@ export const StyledGroupWrapper = styled.div`
   }
 `;
 
-export const StyledChevron = styled(ChevronIcon)`
+export const StyledChevron = styled(DownIcon).attrs({
```

Comment:
I think this will be a breaking change (albeit minor) for the web application, so I will patch it.